templates/glue-workflow.j2

```diff
@@ -287,7 +295,36 @@ Resources:
             "--parquet-bucket": !Ref ParquetBucketName
             "--additional-python-modules": "datacompy~=0.8 flask~=2.0 flask-cors~=3.0"
         {% endfor %}
-      Description: This trigger runs after completion of all JSON to Parquet jobs
+      Description: This trigger runs the compare parquet jobs after completion of all JSON to Parquet jobs
+      Type: CONDITIONAL
+      Predicate:
+        Conditions:
+          {% for dataset in datasets if "Garmin" in dataset["data_type"] %}
+          - JobName: !Sub "${Namespace}-{{ dataset["stackname_prefix"] }}-Job"
+            State: SUCCEEDED
+            LogicalOperator: EQUALS
+          {% endfor %}
+        Logical: AND
+      StartOnCreation: true
+      WorkflowName: !Ref JsonToParquetWorkflow
+
+  GreatExpectationsParquetTrigger:
```

Comment:
We should make the trigger data type specific. That way if one of the JSON to Parquet jobs fails we still run the GX job for those that succeeded. We can use Jinja formatting to create each trigger for us.

Reply:
So essentially we'd have something for the GX jobs that mirrors what is currently going on with the JSON to Parquet jobs in this workflow template then because they are all linked to run one after another?

Reply:
Yeah, sort of. It will look a little funny because each JSON to Parquet job will have a trigger watching it, and that trigger will kick off the respective GX job. That's like 10x the number of triggers we have in our current JSON to Parquet workflow. Since both GX jobs and JSON to Parquet jobs use the processed data bucket for their temporary files, we'll want to watch out for exceeding S3 quotas (which can happen if too many writes are happening on a single bucket).